### PR TITLE
Added function doc string

### LINKED ISSUE #319: Add docstring to the functions in the code
It is not a good practice to not have a docstring in the function and also does not comply with the pip syntax.

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #319

Signed-off-by: yashLadha <201551061@iiitvadodara.ac.in>

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -33,6 +36,11 @@ def index():
 
 
 def generate_badges(_zip=False, _pdf=False):
+    """
+    Function to generate badges
+    :param `_zip` - Flag for checking for generating zip file
+    :param `_pdf` - Flog for checking for generation pdf file
```
Comment (djmgit) [inline app/main.py]: Please fix typo

File: app/generate-badges.py [hunk 2]
```diff
@@ -25,6 +25,12 @@
 
 
 def configure_badge_page(badge_page, options):
+    """
+    Configure the badge page according to the page
+    options as passed in the function
+    :param `badge_page` - Badges directory
```
Comment (djmgit) [inline app/generate-badges.py]: Here badge page is a path to a single badge sheet containing multiple badges, not a directory.

Comment (yashLadha) [reply app/generate-badges.py]: Yaa i am having confusion in some parameters. Thanks for pointing this out. :+1:

**Comment by jajodiaraghav** [comment]
Good work! @yashLadha 
@djmgit Please review

**Comment by vaibhavsingh97** [comment]
@yashLadha Can you please squash your commits into one 😄

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (vaibhavsingh97) [inline .vscode/settings.json]: Can you please remove this? Irrelevant file

**Comment by yashLadha** [comment]
Fine

**Comment by yashLadha** [comment]
@djmgit @vaibhavsingh97 Please review again. It is ready to be merged

**Pull request merged**